### PR TITLE
[WIP] Build package for Chimaera

### PR DESCRIPTION
**Pull request by eniac111**

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -24,7 +24,7 @@ endif
 ifneq (,$(findstring testcases,$(DEB_BUILD_OPTIONS)))
 	DH_OPTIONS=
 else
-	DH_OPTIONS=-Ndsme-testcases
+	DH_OPTIONS=
```
Comment (freemangordon) [inline debian/rules:27]: Why is that needed?

**Comment by MerlijnWajer** [comment]
Closing as we had already done the work in `maemo/beowulf`
